types/react-grid-layout/index.d.ts

```diff
@@ -22,7 +22,7 @@ declare namespace ReactGridLayout {
          * A string corresponding to the component key.
          * Uses the index of components instead if not provided.
          */
-        i?: string;
+        i: string;
```

Comment:
Why? Is this required now?

Reply:
@alitaheri yes, it's required. Please refer to the URL in the PR summary. More details can be found in the library's source repo...

Reply:
@alitaheri Alright, misled by the flow type definition in the source code. Will make it optional and add the `isRearrangeable`.